dockers/docker-orchagent/swssconfig.sh

```diff
@@ -41,6 +41,22 @@ fast_reboot
 
 HWSKU=`sonic-cfggen -d -v "DEVICE_METADATA['localhost']['hwsku']"`
 
+# Don't load json config if system warm start or
+# swss docker warm start is enabled, the data already exists in appDB.
+WARM_START=`redis-cli -n 4 hget "WARM_RESTART|system" enable`
+if [ "$WARM_START" == "true" ]; then
+  exit 0
+fi
+
+WARM_START=`redis-cli -n 4 hget "WARM_RESTART|swss" enable`
```

Comment:
>WARM_RESTART|swss [](start = 33, length = 17)

In the scope of this PR, what is the use case for
1. WARM_RESTART|system == false
2. WARM_RESTART|swss == true

Intuitively, this module should care only the swss one.

Reply:
There are two scenarios:
<1> Individual SWSS docker restart.
either  system == true or swss == true,  warm start will be performed. 

<2> system restart.
It is to be supported. But we may hit issue when system == false, swss == true.   data would have been flushed at system init.  SWSS docker has to perform extra check to make sure data exits for state restore.